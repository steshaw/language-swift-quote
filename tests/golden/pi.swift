let pi = 3.14159 // pi is inferred to be of type Double
let anotherPi = 3 + 0.14159 // anotherPi is also inferred to be of type Double
