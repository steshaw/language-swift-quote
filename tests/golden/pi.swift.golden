let pi = 3.14159
let anotherPi = 3 + 0.14159
